src/SFA.DAS.Employer.PR.Web/Models/PermissionModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using SFA.DAS.Employer.PR.Domain.Models;
+
+namespace SFA.DAS.Employer.PR.Web.Models;
+
+public class PermissionModel
+{
+    public const string PermissionToAddRecordsText = "Yes, employer will review records";
+    public const string PermissionToRecruitReviewAdvertsText = "Yes, employer will review adverts";
+    public const string PermissionToRecruitText = "Yes";
+    public const string NoPermissionToAddRecordsText = "No";
+    public const string NoPermissionToRecruitText = "No";
+    public long Ukprn { get; set; }
+    public string ChangePermissionsLink { get; set; } = null!;
+    public string ProviderName { get; set; } = null!;
+    public string PermissionToAddRecords { get; set; } = null!;
+    public string PermissionToRecruitApprentices { get; set; } = null!;
```

Comment:
These should be nullable as a value is only assigned if permissions are given.

Reply:
No they are always given a value if present.  They are given 'No'